cmd/controller/run.go

```diff
@@ -170,20 +171,22 @@ func Run(opts Options, runLog logr.Logger) error {
 	refTracker := reftracker.NewAppRefTracker()
 	updateStatusTracker := reftracker.NewAppUpdateStatus()
 
-	// initialize deploy factory once - the deploy factory contains a service account token cache which should be only setup once.
-	deployFactory := deploy.NewFactory(coreClient, kcConfig, sidecarCmdExec, runLog)
+	// initialize cluster access once - it contains a service account token cache which should be only setup once.
+	kubeconf := kubeconfig.NewKubeconfig(coreClient, runLog)
+	compInfo := componentInfo.NewComponentInfo(coreClient, kubeconf, Version)
```

Comment:
lol ok i know "real go programmers don't use whole words" or something,  but this definitely feels like i'm suddenly in the HR portal checking on my compensation info.   anyways it's probably unambiguous given the context.